### PR TITLE
Closed previous PR. Didn't import torch. Now it is fixed.

### PR DESCRIPTION
**Pull request by yutaizhou**
Original implementation assumes the edge index belongs to a graph that is not an induced subgraph. New version returns the number of unique node entries in edge_index, rather than the max node label + 1.

### CODE REVIEW

**Comment by rusty1s** [comment]
I cannot accept this PR. The number of nodes still need to include isolated nodes.

**Comment by yutaizhou** [comment]
Gotcha. After closer inspect, I realize that I did not understand the intended use. I must have mistaken the function as something a user can call, rather than for internal use only. Sorry!

**Comment by rusty1s** [comment]
No problem, thank you for all your contributions :)
